app/src/main/res/values-az/strings.xml

```diff
@@ -46,6 +46,12 @@
     <string name="schedule_send_warning">Keep the phone on and make sure there is nothing killing the app while in background.</string>
     <string name="update_message">Update message</string>
     <string name="send_now">Send now</string>
+    <!-- Message details -->
+    <string name="message_details">Message details</string>
+    <string name="message_details_sender">Sender</string>
+    <string name="message_details_receiver">Receiver</string>
+        <string name="message_details_sent_at">Sent at</string>
```

Comment:
why the extra spaces?

Reply:
Not sure how that happened :/ Let me fix it quickly